core/src/main/java/curso/modelo/Valores.java

```diff
@@ -0,0 +1,5 @@
+package curso.modelo;
+
+public enum Valores {
```

Comment:
"Valores" no representa el concepto de Nivel del curso. Debería llamarse Level, Nivel, etc y en singular. Los Enums representan una lista de valores, pero solo uno en un momento.
```suggestion
public enum Nivel {
```